src/strcasecmp.c

```diff
@@ -19,6 +19,7 @@ extern "C"
 
 #include <errno.h>
 #include <string.h>
+#include <strings.h>
```

Comment:
Is this standard?

Reply:
I'll try to figure out what the basis for it is. I just know that I needed to add it to make the build work with Zephyr (at least with zephyr 3.5 and 3.6... maybe earlier versions of zephyr were different).  If it turns out to be Zephyr specific, would you like for it to be wrapped in "if defined(__ZEPHYR__)"?

Reply:
Which is the functionality that requires this include?

Reply:
I was wrong. The addition of <strings.h> was something that one of my team members added for some debugging that they were doing. This change isn't necessary. Would you like for me to just change this on the same branch?

Reply:
Yes, just remove it in this same PR

Reply:
Done.